netmaster/objApi/objapi_test.go

```diff
@@ -617,8 +631,11 @@ func TestTenantDelete(t *testing.T) {
 	checkCreateNetwork(t, false, "tenant1", "net1", "data", "vlan", "10.1.1.1/16", "10.1.1.254", 1, "", "")
 	checkCreateNetwork(t, false, "tenant1", "net2", "data", "vlan", "20.1.1.1/16", "20.1.1.254", 2, "", "")
 	checkCreateEpg(t, false, "tenant1", "net1", "group1", []string{}, []string{})
+	checkInspectEpg(t, false, "tenant1", "group1", 1)
```

Comment:
These tests are a good start.
But, they can not verify if epg-inspect is showing the endpoints correctly.
We should start adding integration tests for these cases to emulate an endpoint create and verify they show up in inspect.
Basic framework for integration tests are part of PR #478. Can we wait for #478 to be merged and add integration tests for epg-inspect as well?